extra/import_data.py

```diff
@@ -1,197 +1,158 @@
-                        if len(data_headers) == 0:  # get headers
-                            data_headers = [col.value for col in row]
-                        else:  # values
-                            variant = {}
-                            for i in range(len(data_headers)):  # iterate headers
-                                key = data_headers[i].replace(" ", "_").replace(".", "\uff0e").lower()
-
-                                # skip everything that starts with father or mother etc since the key
-                                # name contains patient number of parents
-                                # TODO: implement this properly
-                                if key.startswith("father") or key.startswith("mother") \
-                                        or key.startswith("brother") or key.startswith("sister"):
-                                    continue
-
-                                # only use the name inbetween brackets to store, if last char is a right bracket
-                                if key.endswith(")"):
-                                    index_bracket_left = key.rfind("(")
-                                    if index_bracket_left >= 0:
-                                        key = key[index_bracket_left + 1:-1]
-
-                                val = row[i].value
-                                if key == "labels":
-                                    labels = val.split(";")
-                                    val = {}
-                                    for label in labels:
-                                        labelInfo = label.split(",")
-                                        val[labelInfo[0].replace(" ", "_").replace(".", "\uff0e").lower()] = labelInfo[1]
-                                if val is not None and val != "":
-                                    variant[key] = val
-
-                            # add variant info along with metadata to collection
-                            variant.update(patient)
-                            db.insert_one(variant)
-            bar.update(bar.value)
-        logger.debug('finished uploading: %s' % file)
-        # Moves file to other folder, to make uploading a new file (with same filename) possible on the UMCU server (for replacing)
-        os.replace(file, os.path.dirname(__file__) + "/verwerkt/" + os.path.basename(file))
-    print("## \t\tFinished in {0:.2f} minutes".format((time.time() - start_time) / 60))
-
-
-main()
+    '''Main function, for extracting the data from the Alissa output and send to function to parse to MongoDB'''
+     # connection with MongoDB and the correct database "vus" and collection "lastUpdatedOn"
+    mongoclient = pymongo.MongoClient("mongodb://localhost:27017/")
+    db = mongoclient.vus.lastUpdatedOn
+    
+    # Retrieve the last time MongoDB has updated itself, to filter the latest Alissa analyses only
+    lastUpdatedOn_mongoDB = db.find_one({"version" : 1}, {"lastUpdatedOn":1, "_id":0})
```

Comment:
Use `_` to separate words.  I would rename var to: `db_last_updated_on`.

Reply:
check, will do (also for the others you mentioned)